src/shell/lisa_gen_buildroot_rootfs

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -eu
+
+BUILDROOT_URI=git://git.busybox.net/buildroot
+BUILDROOT_VERSION=2018.08
+BUILDROOT_DIR=$LISA_HOME/src/buildroot
```

Comment:
How do we get buildroot in that folder ?

Reply:
we clone it passing it as the path to clone to

Reply:
Yes right... should have cancelled this when I found it out. :)